plugins/recorded_future/komand_recorded_future/actions/download_vulnerability_risk_list/action.py

```diff
@@ -16,12 +18,18 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        query_params = {"format": "xml/stix/1.2", "gzip": "false"}
+        query_params = {"format": "xml/stix/1.2", "gzip": "true"}
```

Comment:
Isn't that the same for all the actions?

Reply:
Only risk list actions, these are query params so may be individual to each action.